bitnami/dataplatform-bp2/README.md

```diff
@@ -0,0 +1,257 @@
+
+This Helm chart enables the fully automated Kubernetes deployment of such multi-stack data platform, covering the following software components:
+
+-   Apache Kafka              – Data distribution bus with buffering capabilities
+-   Apache Spark              – In-memory data analytics
+-   Elasticsearch with Kibana – Data persistence and search
+-   Logstash                  - Data Processing Pipeline
+
+These containerized stateful software stacks are deployed in multi-node cluster configurations, which is defined by the Helm Chart blueprint for this data platform deployment, covering:
+
+-   Pod placement rules – Affinity rules to ensure placement diversity to prevent single point of failures and optimize load distribution
+-   Pod resource sizing rules – Optimized Pod and JVM sizing settings for optimal performance and efficient resource usage
+-   Default settings to ensure Pod access security
+-   Optional Wavefront Observability framework configuration
+
+In addition to the Pod resource optimizations, this blueprint is validated and tested to provide Kubernetes node count and sizing recommendations [(see Kubernetes Cluster Requirements)](#kubernetes-cluster-requirements) to facilitate cloud platform capacity planning. The goal is optimize the number of required Kubernetes nodes in order to optimize server resource usage and, at the same time, ensuring runtime and resource diversity.
+
+The first release of this blueprint defines a small size data platform deployment, deployed on 3 Kubernetes application nodes with physical diverse underlying server infrastructure.
+
+Use cases for this small size data platform setup include: data and application evaluation, development, and functional testing.
+
+## TL;DR
+
+```console
+$ helm repo add bitnami https://charts.bitnami.com/bitnami
+$ helm install my-release bitnami/dataplatform-bp2
+```
+
+## Introduction
+
+This chart bootstraps Data Platform Blueprint-2 deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+The "Small" size data platform in default configuration deploys the following:
+
+1. Zookeeper with 3 nodes to be used for both Kafka
+2. Kafka with 3 nodes using the zookeeper deployed above
+3. Elasticsearch with 3 master nodes, 2 data nodes, 2 coordinating nodes and 1 kibana node
+4. Logstash with 2 nodes
+5. Spark with 1 Master and 2 worker nodes
+
+The data platform can be optionally deployed with the wavefront observability framework. In that case, the wavefront collectors will be set up as a DaemonSet to collect the Kubernetes cluster metrics to enable runtime feed into the Tanzu Observability service. It will also be pre-configured to scrape the metrics from the Prometheus endpoint that each application (Kafka/Spark/Elasticsearch/Logstash) emits the metrics to.
```

Comment:
Shouldn't this be part of the "Configuration and installation details"?

Reply:
I don't think so, that section is at the end of the README while this is information that should be much more visible.
Also, they don't have a webpage yet where this information appears so this is the "platform presentation"

Reply:
Fair enough